android/src/fdroid/play/listings/be/release-notes.txt

```diff
@@ -1,12 +1,12 @@
-• Свежыя даныя OpenStreetMap па стане на 10 ліпеня 2023 г
-• Выпраўлена некалькі збояў
-• Падтрымка вышыні, OsmAnd і колераў Garmin у імпартаваных дарожках GPX
-• Выпраўлены час прыбыцця для пешага маршрута з вялікімі схіламі
-• Кнопка «Абагуліць» была перамешчана на старонку «Месца»
-• Абноўлены пераклады
-• Даданы новыя крамы і тыпы мерапрыемстваў
-• Палепшаны рэндэрынг водных тунэляў і мастоў
-• Больш падрабязныя тыпы чыгункі
-• Даданая boundary=protected_area
+• Новыя дадзеныя OpenStreetMap па стане на 14 жніўня
```

Comment:
```suggestion
• Новыя даныя OpenStreetMap па стане на 14 жніўня
```

Reply:
Should we fix it also in the About dialog?